app/src/main/sentences/sl/navigation.dslf

```diff
@@ -0,0 +1,3 @@
+navigation: high
+(pelji me?)|(pelji me|zapelji me)|((poišči mi?)? navodila<s?>) v|čez|v? .where.;
```

Comment:
The `|` applies to either single words, or pieces of a sentence within parenthesis. So `pelji me|zapelji me` would match "pelji me me" and "pelji zapelji me", which I doubt is what you meant. The `pelji|zapelji me?` I suggested below, instead, matches "pelji", "pelji me", "zapelji", "zapelji me". I also removed the duplicate "v"
```suggestion
(pelji|zapelji me?)|((poišči mi?)? navodila<s?>) v|čez? .where.;
```